<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>businessGroup</class>
 <widget class="QMainWindow" name="businessGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>912</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1181</width>
      <height>821</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1181</width>
       <height>790</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Milestone 1-Page 1</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1181</width>
        <height>791</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <weight>12</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 175, );
color: rgb(0, 0, 0);
background-color: rgb(180, 190, 200);
font: 100 12pt &quot;Times New Roman&quot;;
</string>
      </property>
      <property name="title">
       <string>Business Page</string>
      </property>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>571</width>
         <height>161</height>
        </rect>
       </property>
       <property name="title">
        <string>Select Location</string>
       </property>
       <widget class="QComboBox" name="stateList">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(236, 236, 236);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>State</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>20</y>
          <width>55</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>City</string>
        </property>
       </widget>
       <widget class="QListWidget" name="cityList">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>40</y>
          <width>201</width>
          <height>121</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(236, 236, 236);</string>
        </property>
       </widget>
       <widget class="QListWidget" name="zipcode">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>40</y>
          <width>161</width>
          <height>121</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(236, 236, 236);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>20</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Zip Code</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>200</y>
         <width>1171</width>
         <height>641</height>
        </rect>
       </property>
       <property name="title">
        <string>Filter On Category</string>
       </property>
       <widget class="QListWidget" name="categoryList">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>241</width>
          <height>251</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(236, 236, 236);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Category</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>20</y>
          <width>841</width>
          <height>291</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <weight>12</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 175, );
color: rgb(0, 0, 0);
background-color: rgb(180, 190, 200);
font: 100 12pt &quot;Times New Roman&quot;;
</string>
        </property>
        <property name="title">
         <string>Business</string>
        </property>
        <widget class="QTableWidget" name="businessTable">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>891</width>
           <height>261</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(236, 236, 236);</string>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>310</y>
          <width>531</width>
          <height>271</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 208, 225);</string>
        </property>
        <property name="title">
         <string>Popular Business By Zipcode</string>
        </property>
        <widget class="QTableWidget" name="popularBusinessTable">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>511</width>
           <height>241</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
           <weight>12</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(240, 240, 240);
color: rgb(0, 0, 0);</string>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>309</y>
          <width>541</width>
          <height>271</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777210</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(189, 255, 214);</string>
        </property>
        <property name="title">
         <string>Successful Business By Zipcode</string>
        </property>
        <widget class="QTableWidget" name="successfulBusinessTable">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>521</width>
           <height>241</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
           <weight>12</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(240, 240, 240);
color: rgb(0, 0, 0);</string>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QPushButton" name="loadDataByZipcode">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <weight>12</weight>
          <italic>false</italic>
          <bold>false</bold>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/refresh1.png</normaloff>icons/refresh1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="lookup">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>30</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <weight>12</weight>
          <italic>false</italic>
          <bold>false</bold>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/search.png</normaloff>icons/search.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="clearSearch">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>100</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <weight>12</weight>
          <italic>false</italic>
          <bold>false</bold>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/delete1.png</normaloff>icons/delete1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="zipcodeStat">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>20</y>
         <width>581</width>
         <height>171</height>
        </rect>
       </property>
       <property name="title">
        <string>Zipcode Statistics</string>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string># Of Businesses</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Average Income</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Total Population</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="numOfBusinesses">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>30</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="tabletTracking">
         <bool>true</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="totalPopulation">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>70</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="avgIncome">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>110</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QTableWidget" name="topCategoryTable">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>331</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <weight>12</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(240, 240, 240);</string>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1261</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
